client/blocks/login/error-notice.jsx

```diff
@@ -65,7 +65,7 @@ class ErrorNotice extends Component {
 		 * The existing_wpcom_user error is caught in SocialLoginForm.
```

Comment:
Need to update the comment to note that `user_exists`. Or change it to "These errors are caught..."